src/TextField/TextField.jsx

```diff
@@ -454,7 +465,8 @@ const TextField = React.createClass({
       disabled: this.props.disabled,
       onKeyDown: this._handleInputKeyDown,
     };
-    const inputStyle = this.mergeStyles(styles.input, this.props.inputStyle);
+    inputStyle = this.mergeStyles(styles.input, inputStyle);
+    textareaStyle = this.mergeStyles(styles.textarea, textareaStyle);
```

Comment:
We play to do this change:
```diff
   render() {
-     let {
+     const {
```
Could we create a new variable like `const textareaStyleMerged = `?